src/Bejnarkli.hs

```diff
@@ -46,25 +45,21 @@ bejnarkliServer bs password = do
     else yield $ U8S.fromString "n"
 
 idC :: Monad m => ConduitT i i m ()
-idC = await >>= maybe (return ()) (\a -> yield a >> idC)
+idC = await >>= maybe (pure ()) (\a -> yield a *> idC)
 
 prependSource :: Monad m => i -> ConduitT i i m ()
 prependSource prefix =
   getZipConduit (ZipConduit (yield prefix) *> ZipConduit idC)
 
 parseResponse :: Monad m => ConduitT BS.ByteString Void m Bool
-parseResponse = do
-  response <- await
-  pure $ response == Just (U8S.fromString "y")
+parseResponse = (== Just (U8S.fromString "y")) <$> await
```

Comment:
I don't like giving names to things, unless it really helps with readability. I also like to await `do` notation unless it make the code **much** more ergonomic to write or **much** easier to understand.